src/sc/embedder/content/embedder.py

```diff
@@ -93,7 +94,7 @@ class IEmbedder(form.Schema):
         required=False,
     )
 
-    width = schema.Int(
+    width = schema.TextLine(
         title=_(u'Width'),
         description=_(u''),
```

Comment:
@rodfersou update the description here to reflect the changes and the expected value